e2e_tests/dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.12
+ARG parent_dir
+# this environment variable specifies where playwright should download browsers and ffmpeg, unused for now
+# ENV PLAYWRIGHT_BROWSERS_PATH=/app/pw-browsers
+WORKDIR /e2e_tests
+
+# seems like poetry doesn't pick up pyproject.toml for some unapparent reason - same with the readme, might break if removed
+COPY pyproject.toml pyproject.toml
+RUN touch README.md
```

Comment:
why do you need a blank README in here?

Reply:
was left over from an older version of the file, will remove 🤝